ocaml/tests/test_common.ml

```diff
@@ -191,7 +191,7 @@ let make_vlan ~__context ~tagged_PIF ~untagged_PIF ~tag ?(other_config=[]) () =
 let make_network_sriov = Xapi_network_sriov.create_internal
 
 let make_bond ~__context ?(ref=Ref.make ()) ?(uuid=make_uuid ()) ~master ?(other_config=[]) ?(primary_slave=Ref.null) ?(mode=`activebackup) ?(properties=[]) () =
-  Db.Bond.create ~__context ~ref ~uuid ~master ~other_config ~primary_slave ~mode ~properties ~links_up:0L;
+  Db.Bond.create ~__context ~ref ~uuid ~master ~other_config ~primary_slave ~mode ~properties ~links_up:0L ~auto_update_mac:false;
```

Comment:
Make this an optional argument `?(auto_update_mac=false)`.

Reply:
i can't figure out how to make this argument optional, and could not find an example elsewhere?

(i tried to add an element to the versioned_params list in datamodel.ml, Bond.create, but this didn't work)

Reply:
aah, you mean an optional argument of `make_bond`... :+1: